code/code/misc/info.cc

```diff
@@ -6012,7 +6012,7 @@ void TBeing::doPrayers(const sstring &argument)
         for (comp = 0; (comp < CompInfo.size()) && (i != CompInfo[comp].spell_num);comp++);
 
         if (comp != CompInfo.size() && CompInfo[comp].comp_num >= 0) 
-          sprintf(buf + strlen(buf), "   [%2i] %s",  totalcharges, obj_index[real_object(CompInfo[comp].comp_num)].short_desc); 
+          sprintf(buf + strlen(buf), "   [%2i] %s",  totalcharges, obj_index[real_object(CompInfo[comp].comp_num)].short_desc.c_str()); 
```

Comment:
The issue with this lookup is being addressed in a separate PR that refactors the doPrayers command entirely (and makes it actually work, as currently it's not even fully implemented). Leaving these changes in this PR so that it builds/works successfully as coded. This won't conflict with the other PR.